test/runtests.bat

```diff
@@ -6,8 +6,19 @@ if not exist ..\bin mkdir ..\bin
 REM compile the code into the bin folder
 javac  ..\src\seedu\addressbook\Addressbook.java -d ..\bin
 
+REM run the program and pass in invalid file paths
+java -classpath ..\bin seedu.addressbook.AddressBook " " < NUL > actual.txt
+java -classpath ..\bin seedu.addressbook.AddressBook "directoryThatDoesNotExist/valid.filename" < NUL >> actual.txt
+java -classpath ..\bin seedu.addressbook.AddressBook ".noFilename" < NUL >> actual.txt
+REM run the program and pass in valid a file path
+mkdir data
```

Comment:
Does this still work if the `data` directory does not exist? Or does it show an error?